applications/ExternalSolversApplication/custom_utilities/feast_condition_number_utility.h

```diff
@@ -0,0 +1,250 @@
+//    |  /           |
```

Comment:
Add this to the structural mechanics so that you will not need including feasthere

Reply:
Or to the app in which the feast solver is added

Reply:
It is giving many problems of dependencies, ask @roigcarlo , I think I will remove